dev-docs/transformation-tests.md

```diff
@@ -16,28 +16,36 @@ make transformation_test_update
 ```
 These targets are part of the `precommit` and `precommit_update` targets respectively.
 
+If you are on a new branch that updates Fluent Bit or otelopscol and are locally verifying the transformation tests, you can force the agents to rebuild by deleting the `dist` folder before running `make transfomration_test`; the target will rebuild the agents if they are not present.
+
 Read on if you are interested in how the transformation tests actually work.
 
 ## Usage
 
-The transformation tests take paths to Fluent Bit and otelopscol binaries to execute directly and compare expected output. To run the tests, first you will need to build both binaries. There are scripts to locally build each agent:
+The transformation tests take paths to Fluent Bit and otelopscol binaries to execute directly and compare expected output. To run the tests, first you will need to build both binaries. Use the build scripts to build each agent and install them into the `dist` folder:
 ```bash
-sh ./builds/fluent_bit_local.sh
-sh ./builds/otel_local.sh
+sh ./builds/fluent_bit.sh $(pwd)/dist
+SKIP_OTEL_JAVA=true sh ./builds/otel.sh $(pwd)/dist
 ```
-After running each script, you will have a `dist` folder that contains each binary.
+After running each script, you will have a `dist` folder that contains the full installation of both agents.
```

Comment:
Not quite the full installation, since the OTel Java stuff would be missing… How about this:
```suggestion
After running each script, you will have a `dist` folder that contains the agent binaries at installation paths.
```

Reply:
Done in a similar way in [0db4c79](https://github.com/GoogleCloudPlatform/ops-agent/pull/1722/commits/0db4c799758b739846eb41852b0ef9654fe4f9c5)